libbeat/template/load.go

```diff
@@ -102,6 +103,39 @@ func (l *ESLoader) Load(config TemplateConfig, info beat.Info, fields []byte, mi
 	return nil
 }
 
+// LoadFinalPipeline loads the final pipeline.
+func (l *ESLoader) LoadFinalPipeline(config TemplateConfig, info beat.Info) error {
```

Comment:
I think we are missing a 'good' package for loading pipelines. Maybe we can introduce it to the idxmgmt package instead of template? The idxmgmt package configures `lifecycle.name` and `lifecycle.rollover_alias` when creating the TemplateConfig.